sdk/keyvault/azure-keyvault-administration/tests/test_access_control.py

```diff
@@ -6,45 +6,45 @@
 import time
 import uuid
 
-from azure.keyvault.administration import KeyVaultRoleScope, KeyVaultPermission, KeyVaultDataAction
+import pytest
+from azure.keyvault.administration import KeyVaultDataAction, KeyVaultPermission, KeyVaultRoleScope
+from devtools_testutils import add_general_regex_sanitizer, recorded_by_proxy, set_bodiless_matcher
 
 from _shared.test_case import KeyVaultTestCase
-from _test_case import AdministrationTestCase, access_control_client_setup, get_decorator
-
+from _test_case import KeyVaultAccessControlClientPreparer, get_decorator
 
 all_api_versions = get_decorator()
 
 
-class AccessControlTests(AdministrationTestCase, KeyVaultTestCase):
-    def __init__(self, *args, **kwargs):
-        super(AccessControlTests, self).__init__(*args, match_body=False, **kwargs)
-
+class TestAccessControl(KeyVaultTestCase):
     def get_replayable_uuid(self, replay_value):
         if self.is_live:
             value = str(uuid.uuid4())
-            self.scrubber.register_name_pair(value, replay_value)
             return value
         return replay_value
 
     def get_service_principal_id(self):
         replay_value = "service-principal-id"
         if self.is_live:
             value = os.environ["AZURE_CLIENT_ID"]
-            self.scrubber.register_name_pair(value, replay_value)
```

Comment:
We could probably put a sanitizer back in here, to enable the `principal_id` check in tests again. The `AZURE_CLIENT_ID` at this point will be the same as `KEYVAULT_CLIENT_ID` in the test environment, so we could uncomment the sanitizer for that variable in `conftest.py` and use "service-principal-id" as the replacement value